### ISSUE TITLE
I got inexplicable  results on “mnist_pixel”，I didn't find the reason.

### ISSUE THREAD

**Issue by ziganlan** [philipperemy/keras-tcn#27]
I did not modify any parameters, but I got a val_accuracy about 0.11 till many epochs:
55000/55000 [==============================] - 164s 3ms/step - loss: 2.3037 - accuracy: 0.1120 - val_loss: 2.3015 - val_accuracy: 0.1135

Is there any special setting in this example("mnist_pixel")?

ps: I try other examples too, and the result seems good.

**Comment by ziganlan** [comment]
I got it . The possible reason maybe my input data. The result become good after input data normalization.

**Comment by philipperemy** [comment]
Yeah it should work correctly :)

Happy to hear that it worked!
